roc/version_test.go

```diff
@@ -0,0 +1,13 @@
+package roc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersion_Get(t *testing.T) {
+	v := GetVersion()
+	require.NotEmpty(t, v)
+	require.NotEmpty(t, v.Library)
```

Comment:
We can also test that:

* at least one of the version components is non-zero
* major versions of C and Go are the same
* minor version of Go >= minor version of C

These invariants should be always true.

Reply:
for Go version seems not able to get it during unit testing
only available after it's built
so don't check it

Reply:
btw not sure how to build this, there is no main.go

Reply:
weird, the `roc_get_version` should be definitely available during unit tests, just to smoke test, do you have libroc installed? I wrote a simple sanity test to check it:
```
package main

/*
#cgo LDFLAGS: -lroc

#include <roc/version.h>
*/
import "C"

import (
        "fmt"
)

func main() {
        var cVersion C.struct_roc_version
        C.roc_get_version(&cVersion)
        fmt.Print(cVersion)
}
```
My output is 
```
 go run /tmp/test.go
{0 2 1}%    
```

I think I have an old version of roc where the method for getting the version is named differently, please rename it if it does not work. Let me know if you have any problems with this!

Reply:
> btw not sure how to build this, there is no main.go

there's no main.go because it's a library, basically just a set of functions that can be used by applications that do have main.go. Unit tests pull on those methods (by having a func main declared somewhere under the hood), so if you want to test something, write a test for it with go unit testing framework or make a small program like I did, but don't commit it unless it's in the unit test form.

Reply:
> I think I have an old version of roc where the method for getting the version is named differently, please rename it if it does not work. Let me know if you have any problems with this!

yes in newer version it's renamed https://roc-streaming.org/toolkit/docs/development/changelog.html#version-0-2-2-feb-27-2023

> there's no main.go because it's a library, basically just a set of functions that can be used by applications that do have main.go. Unit tests pull on those methods (by having a func main declared somewhere under the hood), so if you want to test something, write a test for it with go unit testing framework or make a small program like I did, but don't commit it unless it's in the unit test form.

got it, thanks for your explanation!

Reply:
did that small test of mine work for you? if not - let me know, I'd be happy to help :rocket:

Reply:
yes the test works

Reply:
@Asalle 

> weird, the roc_get_version should be definitely available during unit tests

I think what was missing is **bindings** version (which we tried to get using ReadBuildInfo), not libroc version. See https://github.com/roc-streaming/roc-go/pull/61#discussion_r1145008792

Reply:
Oh I see, had to get some context to review this PR properly